website/docs/zh/config/lib/bundle.mdx

```diff
@@ -1 +1,144 @@
 # lib.bundle
+
+- **类型：** `boolean`
+- **默认值：** `undefined`
+
+指定是否打包库，当 `bundle` 设置为 `true` 时称为打包模式，设置为 `false` 时称为 bundleless 模式。更多详情请参见 [bundle / bundleless](/guide/basic/output-structure#bundle--bundleless)。
+
+::: warning
+无打包模式尚未完全支持，某些功能如 [资源](/guide/advanced/assets) 可能无法正常工作。
```

Comment:
```suggestion
bundleless 模式尚未完全支持，某些功能如 [资源](/guide/advanced/assets) 可能无法正常工作。
```

Reply:
I remember I have remove the `/guide/advanced/assets` docs, maybe we should update here in both zh and en docs.

Reply:
The asset page is available only by a link, https://lib.rsbuild.dev/guide/advanced/assets. I think it's fine to leave it there as a placeholder.